server/game/core/event/EventWindow.js

```diff
@@ -60,14 +63,21 @@ class EventWindow extends BaseStepWithPipeline {
         this.events.forEach((event) => event.checkCondition());
     }
 
-    openWindow(abilityType) {
+    /** open a triggering window for any ability type other than {@link AbilityType.Triggered} */
+    openNonTriggeredWindow(abilityType) {
```

Comment:
So this is used for things like constant effects (like snoke or Dodonna?)

Reply:
It’s for replacement effects since those have different timing than regular triggered abilities. Constant abilities do have a window where they might need to be ordered but it’s not in response to a game event like triggered abilities or replacement effects are.

I meant to rename this method to be more specific, will do that.